test/e2e/crud.go

```diff
@@ -77,7 +77,10 @@ var _ = Describe("Federated", func() {
 				testObjectFunc := func(namespace string, clusterNames []string) (*unstructured.Unstructured, error) {
 					return common.NewTestObject(typeConfig, namespace, clusterNames, fixture)
 				}
-				validateCrud(f, tl, typeConfig, testObjectFunc)
+				orphanDependents := false
+				validateCrud(f, tl, typeConfig, testObjectFunc, &orphanDependents)
+				orphanDependents = true
+				validateCrud(f, tl, typeConfig, testObjectFunc, &orphanDependents)
```

Comment:
It is not necessary to validate non-golden path deletion for every single resource.  To do so increases execution cost with negligible increase in test coverage.  It should be sufficient to test orphaning for a single resource type, and to write a test that specifically creates and deletes a resource.   Then the test needs to at least attempt cleanup of the now orphaned resources in member clusters, since the resources will otherwise not be removed when testing against a namespaced deployment.